
Status: in BTS.  need also upstream fixes (merged in cvs)

diff -ur gcc-4.0-4.0ds0.old/debian/rules.defs gcc-4.0-4.0ds0/debian/rules.defs
--- gcc-4.0-4.0ds0.old/debian/rules.defs	2004-11-29 18:54:45.000000000 +0100
+++ gcc-4.0-4.0ds0/debian/rules.defs	2004-11-29 18:58:58.000000000 +0100
@@ -198,7 +198,7 @@
 endif
 
 java_no_cpus := mips mipsel
-java_no_systems := hurd-i386 kfreebsd-gnu knetbsd-gnu netbsd-elf-gnu
+java_no_systems := hurd-i386 knetbsd-gnu netbsdelf-gnu
 ifneq (, $(filter $(DEB_TARGET_GNU_CPU),$(java_no_cpus)))
   with_java := disabled for architecture $(DEB_TARGET_GNU_CPU)
 endif
